runbotics/runbotics-desktop/src/action/jira/jira.utils.ts

```diff
@@ -241,6 +247,59 @@ export const getJiraAllSprintTasks = async <T extends CloudJiraUser>(
     };
 };
 
+export const getJiraAllTaskDetails = async <T extends CloudJiraUser>(
+    input: GetTaskDetailsInput,
+) => {
+    const jql = `key="${input.task}"`.trim();
+
+    let startAt = 0;
+    let issues: SearchIssue<T> | null = null;
+
+    const response = await searchTaskDetails<T>(input, jql, startAt);
+    if (response.total > 0) {
+        issues = response.issues[0];
+    }
+
+    return {
+        issues,
+    };
```

Comment:
to avoid using let I recommend sth like
```
const START_AT = 0;
return await searchTaskDetails<T>(input, jql, START_AT)
     .then(res => res.total > 0 ? res.issues[0] : null)
```